module/updatable_configs/manager_test.go

```diff
@@ -101,4 +111,37 @@ func TestManager_RegisterDurationConfig(t *testing.T) {
 	// should succeed setting correct type
 	err = field.Set("1h")
 	assert.NoError(t, err)
+	assert.True(t, util.CheckClosed(fieldSet))
+}
+
+func TestManager_RegisterIdentifierListConfig(t *testing.T) {
+	mgr := updatable_configs.NewManager()
+
+	// should be able to register config
+	fieldSet := make(chan struct{}) // closed when field is successfully set
+	err := mgr.RegisterIdentifierListConfig("field",
+		func() flow.IdentifierList { return nil },
+		func(_ flow.IdentifierList) error { close(fieldSet); return nil })
+	require.NoError(t, err)
+
+	// should be able to get the field
+	field, ok := mgr.GetField("field")
+	assert.True(t, ok)
+	// field must be parseable by structpb (otherwise admin server will error)
+	_, err = structpb.NewValue(field.Get())
+	require.NoError(t, err)
+
+	// should fail to set incorrect type
+	err = field.Set(struct{}{})
+	assert.Error(t, err)
```

Comment:
can you assert this is specifically a validation error. similar comment for other error cases in this test